config/remotePlugin.js

```diff
@@ -93,13 +94,43 @@ module.exports = {
         },
       },
     },
+    {
+      type: 'console.page/route',
+      properties: {
+        path: '/app-studio/workspace-settings',
+        exact: true,
+        component: {
+          $codeRef: 'WorkspaceSettings',
+        },
+      },
+    },
+    {
+      type: 'console.page/route',
+      properties: {
+        path: '/app-studio/workspace-settings',
+        exact: true,
+        component: {
+          $codeRef: 'WorkspaceSettings',
+        },
+      },
+    },
+    {
+      type: 'core.page/route',
+      properties: {
+        path: '/app-studio/workspace-settings',
+        exact: true,
+        component: {
+          $codeRef: 'WorkspaceSettings',
+        },
+      },
+    },
     {
       type: 'core.page/route',
       properties: {
-        path: '/hacbs',
+        path: '/app-studio/workspace-settings',
```

Comment:
Why did you update HACBS route? Only one route extension is needed here but you now have four.

Reply:
updated